lxd/instance/drivers/driver_qemu.go

```diff
@@ -2512,7 +2512,7 @@ func (d *qemu) addRootDriveConfig(sb *strings.Builder, mountInfo *storagePools.M
 	}
 
 	// If the storage pool is on ZFS and backed by a loop file and we can't use DirectIO, then resort to
-	// unsafe async I/O to avoid kernel hangs when running ZFS storage pools in an image file on another FS.
+	// unsafe async I/O to avoid kernel crash when running ZFS storage pools in an image file on another FS.
```

Comment:
lock up would probably be better here as I associate a kernel crash with a panic